web/src/user_pill.js

```diff
@@ -85,6 +86,7 @@ export function append_person(opts) {
 
     pill_widget.appendValidatedData(pill_data);
     pill_widget.clear_text();
+    compose_recipient.update_placeholder_text();
```

Comment:
I'm simply wondering, if the update_placeholder_text wasn't triggered when adding a person, what was responsible for updating the placeholder previously? Shouldn't we address that part which used to update the placeholder?

Reply:
Yeah good point!

I'm trying to find when it was working. I just tried `96d6fda2a72ef5fb9410bef77f5640f94ff4bd2a` from late April and it's still buggy. I'll try an earlier commit in a bit (it takes so long to reprovision)

Reply:
I'm having trouble getting a provision on 6.0 to work properly. @karlstolley do you have any ideas on how to figure this out?

Reply:
I figured it out! And it was indeed a bug, so thanks for encouraging me to look into it :)

Reply:
Yay! That's cool!